orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -229,6 +231,8 @@ else if (mCurrentStatus.equals(STATUS_ON)) {
         }
 
         startForeground(NOTIFY_ID, mNotifyBuilder.build());
```

Comment:
shouldn't this line be commented out?

Reply:
yikes, good catch. not sure how that happened.

Using ServiceCompat should allow it to work on older devices.